src/docker/Dockerfile

```diff
@@ -119,6 +120,16 @@ RUN wget -O /usr/local/bin/gomplate "https://github.com/hairyhenderson/gomplate/
   && chmod +x /usr/local/bin/gomplate \
   && rm -rf /var/tmp/* /tmp/* /var/tmp/.???* /tmp/.???*
 
+ # Install Podman. Get versions from https://github.com/hairyhenderson/gomplate/releases
+ARG PODMAN_VERSION="3.0.1-2"
+ENV PODMAN_VERSION=${PODMAN_VERSION}
+RUN dnf -y module disable container-tools
+RUN dnf -y install 'dnf-command(copr)'
+RUN curl -L -o /etc/yum.repos.d/devel:kubic:libcontainers:testing.repo https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/testing/CentOS_8/devel:kubic:libcontainers:testing.repo
+RUN dnf -y --refresh install podman
```

Comment:
You need to specify which version of podman to install (using the PODMAN_VERSION variable from line 125).